packages/merge-styles/src/StyleOptionsState.ts

```diff
@@ -0,0 +1,28 @@
+import { IStyleOptions } from './IStyleOptions';
+
+/**
+ * Sets the current RTL value.
+ */
+export function setRTL(isRTL: boolean): void {
```

Comment:
should this be `rtl.ts`?

Reply:
I anticipate this file gathering all "global" state for stylesheets. Another we've talked about adding is a per-component stylesheet in the case of web components.